CMakeLists.txt

```diff
@@ -140,8 +140,9 @@ set(SWIFT_NATIVE_CLANG_TOOLS_PATH "" CACHE STRING
 set(SWIFT_NATIVE_SWIFT_TOOLS_PATH "" CACHE STRING
     "Path to the directory that contains Swift tools that are executable on the build machine")
 
-option(SWIFT_TOOLS_ENABLE_LTO
-  "If set to true, build the swift compiler with link time optimization enabled" FALSE)
+set(SWIFT_TOOLS_ENABLE_LTO OFF CACHE STRING "Build Swift tools with LTO. One
```

Comment:
I'd say something like "This option only affects the tools that run on the host (the compiler), and has no effect on the target libraries (the standard library and the runtime)."